controllers/application_controller.go

```diff
@@ -62,7 +62,7 @@ type ApplicationReconciler struct {
 
 const applicationName = "Application"
 
-//+kubebuilder:rbac:groups=appstudio.redhat.com,resources=applications,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=appstudio.redhat.com,resources=applications,verbs=get;list;watch;create;update;patch;delete;deletecollection
```

Comment:
HAS itself doesn’t need the deletecollection verb, just the RHTAP users, so we don’t need to add it to HAS here